src/mono/wasm/runtime/net6-legacy/hybrid-globalization.ts

```diff
@@ -47,12 +47,20 @@ export function mono_wasm_change_case(exceptionMessage: Int32Ptr, culture: MonoS
     }
 }
 
+function get_utf16_string(ptr: number, length: number): string{
```

Comment:
Is this still used?

Reply:
In chance-case functions. If comparison will turn out to work fine, we can make a future PR that implements this decoding/encoding in change-case as well.